lms/envs/test.py

```diff
@@ -474,9 +474,13 @@
 FEATURES['CERTIFICATES_HTML_VIEW'] = True
 
 # Set dummy values for profile image settings.
-PROFILE_IMAGE_BACKEND = 'django.core.files.storage.FileSystemStorage'
-PROFILE_IMAGE_DOMAIN = 'http://example-storage.com/'
-PROFILE_IMAGE_URL_PATH = 'profile_images/'
+PROFILE_IMAGE_BACKEND_CONFIG = {
+    'storage_class': 'storages.backends.overwrite.OverwriteStorage',
+    'storage_kwargs': {
+        'location': TEST_ROOT / 'uploads/profile_images/',
```

Comment:
in order for this directory to exist under CI, i think this may require a `setUp` call to `os.makedirs` or some other prior arrangement.